server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileCreateDTO.java

```diff
@@ -13,9 +13,21 @@
 @Introspected
 public class MemberProfileCreateDTO {
 
+    @NotBlank
+    @Schema(description = "first name of the employee")
+    private String firstName;
+
+    @Nullable
+    @Schema(description = "middle name of the employee")
+    private String middleName;
+
+    @NotBlank
+    @Schema(description = "last name of the employee")
+    private String lastName;
+
     @Nullable
-    @Schema(description = "full name of the employee")
-    private String name;
+    @Schema(description = "suffix of the employee")
+    private String suffix;
```

Comment:
We need to keep "name" in our dtos and set it using your convenience method. Otherwise the frontend has to append the name together everywhere and that will be a pain.

Reply:
Yeah, I was wondering if this would entail changes to the front-end. This is a create DTO which is sent by the front-end, right. Do you mean it should contain both the full name and its components (first, middle, last name, and suffix)?

Reply:
Yes, exactly.